src/chat-client/index.ts

```diff
@@ -411,6 +411,23 @@ class Chat extends ClientBase {
     }
   }
 
+  /**
+   * Gets device information for a given username.
+   * This method allows you to see device name, description, type (desktop
+   * or mobile), and creation time for all active devices of a given username.
+   * @param username - the Keybase username to get devices for
+   * @example
+   * bot.chat.getDeviceInfo(username).then((devices) => console.log(devices))
+   */
+  public async getDeviceInfo(username: string): Promise<chat1.GetDeviceInfoRes> {
+    await this._guardInitialized()
+    const res = await this._runApiCommand({apiName: 'chat', method: 'getdeviceinfo', options: {username}})
+    if (!res) {
+      throw new Error('Keybase chat get device info returned nothing.')
+    }
+    return res
+  }
+
```

Comment:
Looks OK, you probably should revert the README change, the current docs generation is quite broken. cc @nathunsmitty

Reply:
@pzduniak Could you tell me the version of npm `documentation` you used?  It's not in `devDependencies`.  Maybe if I match that it'll look better.

Reply:
It's been broken ever since we migrated to TS. I tried to fix it in #177 but gave up.

Reply:
When documentation generation is fixed, if we still want to keep this hidden from the docs, we'll want to add an `@ignore` to the jsdoc here so that it continues to ignore this function.

Reply:
It's fine, keeping it hidden was acceptable but not optimal.  I'll merge in now!